static/src/stylesheets/module/site-messages/_fiv-banner.scss

```diff
@@ -122,7 +123,7 @@
 
 .fiv-banner__cta {
 
-    @include mq(tablet) {
+    @include mq(leftCol) {
```

Comment:
You have two `@include mq(leftCol) {` 😄 
Could this block be combined with the one on line 138?

Reply:
good point I'll fix

Reply:
I didn't realise I asked commercial dev for a review, that was a slip, but thanks for taking a look! 😆

Reply:
@johnduffell You didn't.  See https://github.com/guardian/frontend/blob/master/.github/CODEOWNERS